itwin-platform-access/imodels-access-backend/package.json

```diff
@@ -36,10 +36,10 @@
     "axios": "~0.21.1"
   },
   "devDependencies": {
-    "@itwin/core-backend": "^3.6.0",
-    "@itwin/core-bentley": "^3.6.0",
-    "@itwin/core-common": "^3.6.0",
-    "@itwin/core-geometry": "^3.6.0",
+    "@itwin/core-backend": "^3.6.0 || ^4.0.0",
```

Comment:
dont change dev deps

Reply:
if we change the direct dependencies of imodels-access-backend-tests, then these have to change to match other wise rush does not like it, so change back those as well?